datalad/support/json_py.py

```diff
@@ -50,21 +50,28 @@
 from ..dochelpers import exc_str
 
 
-def dump(obj, fname):
+def dump(obj, fname, compressed=False):
+
+    _open = LZMAFile if compressed else open
```

Comment:
Wasn't io.open instead of playing open used for some Unicode compatibility on py2? Ie I just wonder if there was a reason not warranting this change

Reply:
Ah indeed, PY2... Adjusted....